pkg/policy/consumer.go

```diff
@@ -24,6 +24,32 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// TrafficDirection specifies the directionality of policy (ingress or egress).
```

Comment:
Mentioned above, but it seems like this trafficdirection stuff is fairly closely tied to the BPF policymap, so if it were in the policymap package then it could probably be used by all the relevant pieces without introducing any additional package dependencies